includes/Admin/Editor.php

```diff
@@ -433,6 +437,8 @@ public function get_editor_settings(): array {
 			'version'                 => WEBSTORIES_VERSION,
 			'nonce'                   => $nonce,
 			'showMedia3p'             => true,
+			'globalAutoAdvance'       => (bool) $auto_advance,
+			'globalPageDuration'      => $page_duration,
```

Comment:
Should this be cast to an int?

Reply:
Isn't it guaranteed to be an int since we use `register_setting()` with `type => number`?

Reply:
Sadly no. That type does not mean that WordPress casts it for you. That type is used for the REST API, where type is important. 

I var_dumped it and returned a string. 
<img width="130" alt="Screenshot 2022-10-13 at 18 40 46" src="https://user-images.githubusercontent.com/237508/195667771-fc5af0df-8ff3-4111-9c7a-50066ff73be8.png">

WordPress 🙃

Reply:
Thanks for testing! Let's add a cast then.